lib/datasource/packagist/index.ts

```diff
@@ -204,10 +204,10 @@ async function getAllPackages(regUrl: string): Promise<AllPackages | null> {
 
 function getAllCachedPackages(regUrl: string): Promise<AllPackages | null> {
   const cacheKey = `packagist-${regUrl}`;
-  const cachedResult = memCache.get(cacheKey);
+  const cachedResult = memCache.get<AllPackages>(cacheKey);
   // istanbul ignore if
   if (cachedResult) {
-    return cachedResult as Promise<AllPackages | null>;
+    return Promise.resolve(cachedResult);
```

Comment:
I think `cachedResult`is already a `Promise`? 🤔

Reply:
Yes. I'm not super confident with returning the `Promise` instead of the content, but I'll fix the types on this one for now